frontend/app_flowy/packages/appflowy_editor/lib/src/render/color_menu/color_menu.dart

```diff
@@ -0,0 +1,291 @@
+import 'package:appflowy_editor/src/editor_state.dart';
+import 'package:appflowy_editor/src/infra/flowy_svg.dart';
+import 'package:appflowy_editor/src/render/style/editor_style.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flowy_sdk/protobuf/flowy-grid/select_type_option.pb.dart';
+import 'package:app_flowy/plugins/grid/presentation/layout/sizes.dart';
+import 'package:flowy_infra_ui/style_widget/text.dart';
+import 'package:flowy_infra_ui/widget/spacing.dart';
+import 'package:flowy_infra_ui/style_widget/scrolling/styled_list.dart';
+import 'package:app_flowy/plugins/grid/application/field/type_option/edit_select_option_bloc.dart';
```

Comment:
Can't import the additional packages. Because appflowy_editor doesn't depend on them.

1. package:flowy_sdk
2. package:app_flowy

Reply:
These packages are for colours list. Should I create the new list of with name and then use that ?